python/tvm/relay/build_module.py

```diff
@@ -102,6 +102,7 @@ def __init__(self):
         self._get_params_func = self.mod["get_params"]
         self._get_function_metadata = self.mod["get_function_metadata"]
         self._get_devices = self.mod["get_devices"]
+        self._get_irmodule = self.mod["get_irmodule"]
```

Comment:
Any chance to disambiguate get_module vs get_irmodule here -- without doing a breaking change to relay.build ?

Reply:
Unsure what you mean, this does get the `IRModule` map?

Reply:
Also, this is ambiguous all the way through the call stack:
https://github.com/apache/tvm/blob/13f54e03fd3b7066d9ce9b0b0c3e1ba297d627bb/src/relay/backend/aot_executor_codegen.cc#L891

Which I think should be a wider fix to unify `IRModule` rather than in this PR :crying_cat_face:

Reply:
I see; I thought we are implementing that in this PR.